packages/components/src/dimension-control/types.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Internal dependencies
+ */
+import type { IconType } from '../icon';
+
+export type Size = {
+	/**
+	 * Human-readable name of the size.
+	 */
+	name: string;
+	/**
+	 * Short unique identifying name of the size.
+	 */
+	slug: string;
+};
+
+export type DimensionControlProps< IconProps = unknown > = {
+	/**
+	 * Label for the control.
+	 */
+	label: string;
+	/**
+	 * An array of sizes to choose from.
+	 */
+	sizes?: Size[];
```

Comment:
This is definitely nitpicking, and definitely not a blocker for this PR

The `sizes` prop has a default value, but it's quite complex. I wonder if we could do something like

```suggestion
	/**
	 * An array of sizes to choose from.
	 *
	 * @default DEFAULT_SIZES
	 *
	 * @see packages/components/src/dimension-control/sizes.ts
	 */
	sizes?: Size[];
```

@mirka have we ever found ourselves in a similar situation before?

Reply:
Yes, and I think your suggestion is a reasonable solution.

Though, at a deeper level I don't really care because this component isn't used anywhere and doesn't seem to belong in the components package to begin with. We should probably put it on the chopping block for deprecation?

Reply:
> Yes, and I think your suggestion is a reasonable solution.

Cool, let's go with that!

> We should probably put it on the chopping block for deprecation

Agreed. We can first check with other folks (especially folks working on block controls and toolspanel) to make sure that it's a good assumption